Source/stores.cpp

```diff
@@ -2382,7 +1973,7 @@ void StartStore(talk_id s)
 			break;
 		}
 		if (hasAnyItems)
-			StartSmithBuy();
+			StartBuy();

```

Comment:
`StartBuy` already checks if items are available.
So could we remove `hasAnyItems` and change this to `if (!StartBuy())`?